common/dynamicconfig/constants.go

```diff
@@ -31,937 +31,1437 @@ func (k Key) String() string {
 	TransferProcessorFailoverMaxPollRPS
 	// TransferProcessorMaxPollRPS is max poll rate per second for transferQueueProcessor
+	// KeyName: history.transferProcessorMaxPollRPS
+	// Default value: 20
 	TransferProcessorMaxPollRPS
 	// TransferTaskWorkerCount is number of worker for transferQueueProcessor
+	// KeyName: history.transferTaskWorkerCount
+	// Default value: 10
 	TransferTaskWorkerCount
 	// TransferTaskMaxRetryCount is max times of retry for transferQueueProcessor
+	// KeyName: history.transferTaskMaxRetryCount
+	// Default value: 100
 	TransferTaskMaxRetryCount
 	// TransferProcessorCompleteTransferFailureRetryCount is times of retry for failure
+	// KeyName: history.transferProcessorCompleteTransferFailureRetryCount
+	// Default value: 10
 	TransferProcessorCompleteTransferFailureRetryCount
-	// TransferProcessorMaxPollInterval max poll interval for transferQueueProcessor
+	// TransferProcessorMaxPollInterval is max poll interval for transferQueueProcessor
+	// KeyName: history.transferProcessorMaxPollInterval
+	// Default value: 1*time.Minute
 	TransferProcessorMaxPollInterval
 	// TransferProcessorMaxPollIntervalJitterCoefficient is the max poll interval jitter coefficient
+	// KeyName: history.transferProcessorMaxPollIntervalJitterCoefficient
+	// Default value: 0.15
 	TransferProcessorMaxPollIntervalJitterCoefficient
 	// TransferProcessorSplitQueueInterval is the split processing queue interval for transferQueueProcessor
+	// KeyName: history.transferProcessorSplitQueueInterval
+	// Default value: 1*time.Minute
 	TransferProcessorSplitQueueInterval
 	// TransferProcessorSplitQueueIntervalJitterCoefficient is the split processing queue interval jitter coefficient
+	// KeyName: history.transferProcessorSplitQueueIntervalJitterCoefficient
+	// Default value: 0.15
 	TransferProcessorSplitQueueIntervalJitterCoefficient
 	// TransferProcessorUpdateAckInterval is update interval for transferQueueProcessor
+	// KeyName: history.transferProcessorUpdateAckInterval
+	// Default value: 30*time.Second
 	TransferProcessorUpdateAckInterval
 	// TransferProcessorUpdateAckIntervalJitterCoefficient is the update interval jitter coefficient
+	// KeyName: history.transferProcessorUpdateAckIntervalJitterCoefficient
+	// Default value: 0.15
 	TransferProcessorUpdateAckIntervalJitterCoefficient
 	// TransferProcessorCompleteTransferInterval is complete timer interval for transferQueueProcessor
+	// KeyName: history.transferProcessorCompleteTransferInterval
+	// Default value: 60*time.Second
 	TransferProcessorCompleteTransferInterval
 	// TransferProcessorMaxRedispatchQueueSize is the threshold of the number of tasks in the redispatch queue for transferQueueProcessor
+	// KeyName: history.transferProcessorMaxRedispatchQueueSize
+	// Default value: 10000
 	TransferProcessorMaxRedispatchQueueSize
-	// TransferProcessorEnableValidator indicates whether validator should be enabled for transferQueueProcessor
-	TransferProcessorEnableValidator
```

Comment:
nit: maybe we can preserve the original order if that doesn't cost too much effort?

Reply:
Good catch. I accidently moved this one. will  move it back